.github/workflows/test.yml

```diff
@@ -245,7 +245,7 @@ jobs:
             go.mod
             go.sum
       - name: Get data from build cache
-        uses: actions/cache@v3
+        uses: actions/cache@v2
```

Comment:
sorry, but why are we downgrading?

Reply:
Just experimenting to see if I can get CI passing